.release-notes/4152.md

```diff
@@ -0,0 +1,19 @@
+## Support void* (Pointer[None]) parameters in bare lambdas and functions
+
+Unlike conventional FFI functions, bare lambdas `@{(...)}` and bare functions `fun @bare() => ...` did not support Pointer[None] parameters, Pony's equivalent of `void*`. This is despite the fact that like FFI functions, bare lambdas and functions are strictly intended for use with FFI calls and callbacks.
+
+This commit allows bare lambdas and functions to accept `Pointer[None]` parameters which can accept any Pony Pointer[A] type. Therefore, code like the following now works:
```

Comment:
I'd probably rewrote this a bit. Perhaps something like "... and bare functions with `Pointer[None]` parameters to accept arguments of any `Pointer[A]` type."

Reply:
Yes, poor English -- tweaked!